plugins/headlamp-plugin/test-headlamp-plugin.sh

```diff
@@ -53,3 +53,10 @@ npm run lint-fix
 
 # test type script error checks
 npm run tsc
+
+# test upgrade adds missing files
+rm jsconfig.json tsconfig.json src/headlamp-plugin.d.ts
+node bin/headlamp-plugin.js upgrade --no-package-updates
+stat jsconfig.json
+stat tsconfig.json
+stat src/headlamp-plugin.d.ts
```

Comment:
I wonder if it's possible to add a test for a headlamp-plugin version upgrade.

Reply:
I added a test for this.